hubstorage/resourcetype.py

```diff
@@ -40,7 +44,7 @@ def apiget(self, _path=None, **kwargs):
         return self.apirequest(_path, method='GET', **kwargs)
```

Comment:
I disagree, we should set the `GET` default as late as possible. Direct calls to `_iter_lines` also benefits from the retrying without the user duplicating the `setdefault` code. (in this case `iter_json`).

I fixed the `apidelete()` with `kwargs.setdefault`, thanks!